sr_armv7l/make_mode_sp.h

```diff
@@ -35,3 +37,28 @@
     *code++ = 0x2f;						\
     *code++ = 0xe1;						\
     }
+#else	/* __armv6l__ */
+#define GEN_CALL(func)  {					\
+    assert(SIZEOF(*code) == SIZEOF(char));			\
+    /* ldr r0, [pc] */						\
+    *code++ = 0;						\
```

Comment:
A future task here and other place would be to macroize some of these instructions so they can be stored 4 bytes at a time instead of one at a time to speed up compiles.

Reply:
Sure Steve. Can you create an issue for that (since you best understand the requested change) so we track it. Thanks.